packages/schema/src/config/common.ts

```diff
@@ -20,6 +21,18 @@ export default defineUntypedSchema({
    */
   extends: null,
 
+  /**
+   * Specify a compatibility date for your app.
+   *
+   * This is used to control the behavior of presets in Nitro, Nuxt Image
+   * and other modules that may change behavior without a major version bump.
+   *
+   * We plan to improve the tooling around this feature in the future.
+   *
+   * @type {string | Record<string, string>}
+   */
+  compatibilityDate: formatDate(new Date()),
```

Comment:
- I think here defaulting to undefined would be the best so we know user either set it or not and later add prompt
- formatDate maybe can be in `$resolve`?
- comptax has `DateString` we can use to add type checks for valid input